app/javascript/components/time_table/timesheet/timesheet.js

```diff
@@ -57,14 +57,15 @@ class Timesheet extends React.Component {
 
   render() {
     const { projects, tags } = this.state;
+    const projectsForEntries = projects.filter((project) => !['Vacation', 'ZKS'].includes(project.name));
```

Comment:
let's use the accounting? method from project to avoid more name matching conditions. On the backend we need to adjust this accounting? to return true only for Vaction and ZKS.